src/routes/teams-admin.js

```diff
@@ -45,8 +45,58 @@ function register(server, options) {
         handler: getAllTeams
     });
 
+    async function getAllTeamsByUser(request, h) {
+        const { query } = request;
+        const user = await User.findOne({
+            where: {
+                id: query.userId
+            },
+            include: [
+                {
+                    model: Team,
+                    attributes: {
+                        exclude: ['deleted']
+                    },
+                    include: [
+                        {
+                            model: User
+                        }
+                    ]
+                }
+            ]
+        });
+        return {
+            list: user.teams.map(({ dataValues }) => {
+                const { user_team, settings, users, ...data } = dataValues;
+                const owner = users.filter(user => user.user_team.team_role === 'owner').pop();
+                const team = camelizeKeys({
+                    ...data,
+                    memberCount: users.length,
+                    teamRole: user_team.team_role,
```

Comment:
```suggestion
                    team: user_team.team_role,
```
Maybe we can reduce this to `team`. The endpoint already implies this is team information. We don't use `teamMemberCount`, `teamOwner` or `teamId`.

Reply:
the possible values for `teamRole` are `owner`, `admin` and `member`. Wouldn't `team: "member"` be a little confusing?

Reply:
🤦‍♂ I mean **`role`** not `team`. Sorry